projects/fsastorefront-e2e-cypress/cypress/integration/checkout/insurance/change-request.e2e-spec.ts

```diff
@@ -67,6 +67,7 @@ context('Change Request for new user', () => {
     checkout.clickContinueButton();
     checkout.placeOrderOnFinalReview();
     checkout.checkOrderConfirmation();
+    cy.wait(200000);
```

Comment:
added waits due to the order split changes 
- once we have stable checkouts, can be removed for some tests